org.neverball.Neverball.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.neverball.Neverball
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: neverball
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --persist=.neverball
+modules:
+  - name: physfs
```

Comment:
Misses `cleanup` to delete static library and headers.

Reply:
Oops, I forgot adding it. Thanks for reminding me! :-)